src/d3a/setup/settlement_market/default_2a_settlement.py

```diff
@@ -0,0 +1,76 @@
+"""
```

Comment:
Why is the file called `default_2a_settlement.py`? What's `2a`?

Reply:
The default_2a setup file is the setup we alway us for sanity checking, is has been there since the very beginning of d3a and if you have a look, there are multiple adaptions of this setup for different external settings like market type. I thought to stick to the practice but to append the special setting. Will rename it to something more explanatory.

Reply:
Maybe we could rename the `default_2a` file to something else (and later use different suffixes as you did here). I think the main issue is the base name we already had, not your adaptation in the context of this PR.